pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -85,23 +85,23 @@
 , withV4l2M2m ? withV4l2
 , withVaapi ? withHeadlessDeps && (with stdenv; isLinux || isFreeBSD) # Vaapi hardware acceleration
 , withVdpau ? withSmallDeps # Vdpau hardware acceleration
-, withVidStab ? withFullDeps # Video stabilization
-, withVmaf ? withFullDeps && withGPLv3 && !stdenv.isAarch64 # Netflix's VMAF (Video Multi-Method Assessment Fusion)
-, withVoAmrwbenc ? withFullDeps # AMR-WB encoder
+, withVidStab ? withFullDeps && withGPL # Video stabilization
```

Comment:
Vidstab is MIT AFAICT?

Reply:
ffmpeg has it under gpl and looking at theire source it also says gpl https://github.com/georgmartius/vid.stab/blob/master/LICENSE

Reply:
Ah, I was looking at the python module with the same name.